lib/backend.js

```diff
@@ -578,7 +591,16 @@ Backend.prototype._query = function(agent, request, callback) {
   var backend = this;
   request.db.query(request.collection, request.query, request.fields, request.options, function(err, snapshots, extra) {
     if (err) return callback(err);
-    backend._sanitizeSnapshots(agent, request.snapshotProjection, request.collection, snapshots, backend.SNAPSHOT_TYPES.current, function(err) {
+    var requestContext = {
+      requestMethod: 'query',  // TODO: Should this distinguish between queryFetch and querySubscribe?
```

Comment:
Question here for reviewers - This currently doesn't distinguish between "queryFetch" and the initial query issued by "querySubscribe". Should it?

The initial "querySubscribe" DB query is for all intents and purposes identical to the query issued by "queryFetch". The big difference is in whether a pub/sub subscription is registered.

Reply:
yes, let's make the method be the same name as the public method on Backend,  so `queryFetch` or `querySubscribe`